index.js

```diff
@@ -99,12 +100,29 @@ For example:
       `❌ Found an empty lcov file at "${path}".
 An empty lcov file was found but with no coverage data. This might be because \
 you have no test files or your tests are not generating any coverage data.
-`
+`,
     );
     return false;
   }
 
   return true;
 }
 
+function parseMinCoverage(input) {
+  if (input === '') {
+    return 100;
+  }
+
+  const minCoverage = Number(input);
+
+  if (isNaN(minCoverage) || minCoverage < 0 || minCoverage > 100) {
```

Comment:
This:
```js
if (!(minCoverage >= 0 && minCoverage <= 100))
```
is also possible, but may lead to confusion. We cannot refactor to this very natural form:
```js
if (minCoverage < 0 || minCoverage > 100)
```
because `minCoverage` can be `undefined`, and in this case the behavior is not the same.